packages/coinstac-ui/app/render/state/ducks/consortia-page.js

```diff
@@ -0,0 +1,21 @@
+const SET_EXPANDED_COMPUTATION = 'SET_EXPANDED_COMPUTATION';
```

Comment:
This is fine for now, but doesn’t account for multiple expanded computations across multiple consortia pages. We likely want to track this state at some point so users return to UI in a familiar condition.